backend/x11/backend.c

```diff
@@ -188,6 +188,24 @@ static void init_atom(struct wlr_x11_backend *x11, struct wlr_x11_atom *atom,
 	atom->reply = xcb_intern_atom_reply(x11->xcb_conn, atom->cookie, NULL);
 }
 
+
+static void parse_xcb_setup(struct wlr_output *output, xcb_connection_t *xcb_conn) {
+	const xcb_setup_t *xcb_setup;
+	unsigned int length;
```

Comment:
We're not using C89, so putting your declarations at the front isn't necessary. You can just declare them you use use them.